kbase/narrative
# DATAUP-313 table

# Description of PR purpose/changes
This PR is for the job status table. I had to remove quite a bit of the copy pasta code from when I created the container initially for AJ and I to work on. Because I used datatables, the sorting and pagination functionality is included in this PR.

The end result is a simple table: 
![Screen Shot 2020-12-15 at 12 15 59 PM](https://user-images.githubusercontent.com/56279459/102274528-83c86480-3ed0-11eb-998e-ed69a3de579d.png)


# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-313

# Testing Instructions
Navigate to the job status tab. You will be able to sort and see the pagination functionality (provided by data tables). 

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
